src/main/webapp/scripts/script.js

```diff
@@ -8,3 +18,41 @@ function updateDom(text, containerId) {
 function getDomValue(domId) {
   return document.getElementById(domId).value;
 }
+
+function MenuItem (name, link, icon) {
```

Comment:
It will be clearer that is meant to be used with `new` if you make this a `class`

I think it would also be reasonable to do away with this type altogether, and use raw objects for menu items, as in

```
const MENU_ITEMS = [
  {name: 'Trending', link: 'popular.html', icon: 'whatshot'},
  //...
}
```

Reply:
I went with the raw objects option because I didn't want to make a class -> put it in a new file -> load that script or have a class somewhere in the middle of all these constants and functions and whatnot